lib/paths/find-lib.js

```diff
@@ -0,0 +1,16 @@
+import { resolve, dirname, join } from 'path';
```

Comment:
So I believe this is a bit more generic where we need to basically lookup from the main file. For example I used this in the ember build like this. https://github.com/emberjs/ember.js/blob/master/broccoli/packages.js#L87